File: backend2/DataAccess/DaoGenerated/システム共有/tt_afrpthakkorirekiDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             帳票発行履歴テーブル
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afrpthakkorirekiDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afrpthakkorireki";
		public long hakkoseq { get; set; }                        //発行シーケンス
		public int hakkoseqeda { get; set; }                      //発行シーケンス枝番
		public long tyusyutuseq { get; set; }                     //抽出シーケンス
		public string nendo { get; set; }                         //年度
		public string taisyocd { get; set; }                      //抽出対象CD
		public string gyoumucd { get; set; }                      //業務コード
		public string rptyosikihakkocd { get; set; }              //帳票様式発行コード
		public DateTime hakkodttm { get; set; }                   //発行日時
		public string hakkokbn { get; set; }                      //発行区分
		public string? hakkobasyo { get; set; }                   //発行場所
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afrpthakkorirekiDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             帳票発行履歴テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afrpthakkorirekiDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afrpthakkorireki"
        Public Property hakkoseq As Long                        '発行シーケンス
        Public Property hakkoseqeda As Integer                      '発行シーケンス枝番
        Public Property tyusyutuseq As Long                     '抽出シーケンス
        Public Property nendo As String                         '年度
        Public Property taisyocd As String                      '抽出対象CD
        Public Property gyoumucd As String                      '業務コード
        Public Property rptyosikihakkocd As String              '帳票様式発行コード
        Public Property hakkodttm As Date                   '発行日時
        Public Property hakkokbn As String                      '発行区分
        Public Property hakkobasyo As String                   '発行場所
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
